### PR TITLE
DM-29022: Added command line parameters for specifying the idl and sa…

### PR DESCRIPTION
**Pull request by wvreeven**
…lobj versions.

### CODE REVIEW

File: doc/conda/jenkins-conda.rst [hunk 1]
```diff
@@ -22,7 +22,15 @@ Also to find the configuration repo, look for the :meth:`get_config_pkg` in the
...
+.. prompt:: bash
+
+    docker run -e {config_repo}=/path/to/config/repo \\
+        -e idl_version="" -e salobj_version="" \\
+        -v /path/to/your/repos:/home/saluser/develop \\
+        -it lsstts/conda_package_builder
+
 Then run the following inside of the container.
 
 .. prompt:: bash
```
Comment (couger01) [inline doc/conda/jenkins-conda.rst:36]: The last line in this prompt needs to become `conda build --variants "{salobj_version: '', idl_version: ''}' --prefix-length 100 .`. No need for the environment variables in the previous prompt.

**Review by couger01 — CHANGES_REQUESTED** [review]
Just one change and then this looks good to go.

File: doc/conda/jenkins-conda.rst (hunk 1 above)
Comment (wvreeven) [reply doc/conda/jenkins-conda.rst:35]: Thanks, that's very useful. I pushed a fixed version of the file already.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**